SU-SWS/stanford_profile
# Added administrative file list and delete actions

# READY FOR REVIEW

# Summary
- Added missing `/admin/content/files` and added bulk operations to that list.

# Need Review By (Date)
- 1/28

# Urgency
- medium

# Steps to Test
1. checkout this branch
2. `drush cim -y`
3. visit `/admin/content/files` and verify you see the files list.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
